File: RepoTransBench/source_projects/Java/microsoft_vscode-remote-try-java/src/test/java/com/mycompany/app/AppTest.java
package com.mycompany.app;

import org.junit.jupiter.api.*;

import static org.junit.jupiter.api.Assertions.*;

public class AppTest {

    @Test
    public void testGetMessage() {
        assertEquals("Hello Remote World!", App.getMessage());
    }

    @Test
    public void testEvaluateNumber_PositiveEven() {
        assertEquals("Positive Even", App.evaluateNumber(2));
    }

    @Test
    public void testEvaluateNumber_PositiveOdd() {
        assertEquals("Positive Odd", App.evaluateNumber(3));
    }

    @Test
    public void testEvaluateNumber_Negative() {
        assertEquals("Negative", App.evaluateNumber(-7));
    }

    @Test
    public void testEvaluateNumber_Zero() {
        assertEquals("Zero", App.evaluateNumber(0));
    }
}

File: RepoTransBench/target_projects/Java/Go/microsoft_vscode-remote-try-java/tests/original/app_test.go
package tests

import (
	"testing"

	"github.com/mycompany/app/app"
)

func TestGetMessage(t *testing.T) {
	expected := "Hello Remote World!"
	got := app.GetMessage()
	if got != expected {
		t.Errorf("GetMessage() = %q; want %q", got, expected)
	}
}

func TestEvaluateNumber_PositiveEven(t *testing.T) {
	expected := "Positive Even"
	got := app.EvaluateNumber(2)
	if got != expected {
		t.Errorf("EvaluateNumber(2) = %q; want %q", got, expected)
	}
}

func TestEvaluateNumber_PositiveOdd(t *testing.T) {
	expected := "Positive Odd"
	got := app.EvaluateNumber(3)
	if got != expected {
		t.Errorf("EvaluateNumber(3) = %q; want %q", got, expected)
	}
}

func TestEvaluateNumber_Negative(t *testing.T) {
	expected := "Negative"
	got := app.EvaluateNumber(-7)
	if got != expected {
		t.Errorf("EvaluateNumber(-7) = %q; want %q", got, expected)
	}
}

func TestEvaluateNumber_Zero(t *testing.T) {
	expected := "Zero"
	got := app.EvaluateNumber(0)
	if got != expected {
		t.Errorf("EvaluateNumber(0) = %q; want %q", got, expected)
	}
}